config/locales/en.yml

```diff
@@ -23,6 +23,8 @@ en:
   common:
     translations: "Translations"
   components:
+    attachment:
+      opendocument: "This file is in an <a href='https://www.gov.uk/guidance/open-document-format-odf-guidance-for-uk-government/overview-of-productivity-software' target='_blank' class='govuk-link'>OpenDocument</a> format"
```

Comment:
If you suffix this with _html you won't need to run raw on it: https://guides.rubyonrails.org/i18n.html#using-safe-html-translations

Reply:
Should this always be target blank by the way? doesn't look like Whitehall one is: https://github.com/alphagov/whitehall/blob/7b5c5a086b89cb62ffba62b152a0a8dcfc10c8e6/config/locales/en.yml#L447

Probably should use the target attribute to component to decide this, maybe this could be pushed into an argument of this but default to _self?

Reply:
I've changed the target attributes to default to`_self` and the translation is now suffixed with `-html`.